migration/migration/008-install-uda.sql

```diff
@@ -43,6 +51,24 @@ $$
 LANGUAGE plpgsql;
 GRANT EXECUTE ON FUNCTION _prom_catalog.is_timescaledb_oss() TO prom_reader;
 
+CREATE OR REPLACE FUNCTION _prom_catalog.is_multinode()
+    RETURNS BOOLEAN
+AS $func$
+DECLARE
+    is_distributed BOOLEAN = false;
+BEGIN
+    IF _prom_catalog.get_timescale_major_version() >= 2 THEN
+        SELECT count(*) > 0 FROM timescaledb_information.data_nodes
+            INTO is_distributed;
+    END IF;
+    RETURN is_distributed;
+EXCEPTION WHEN SQLSTATE '42P01' THEN -- Timescale 1.x, never distributed
```

Comment:
How can we end up here if we check Timescale version first?

Reply:
i don't know. i actually copied this code, from where it was already implemented, but I agree it shouldn't be possible, especially now that we don't support 1.x anymore